src/ucs/sys/string.h

```diff
@@ -324,6 +324,18 @@ const char* ucs_mask_str(uint64_t mask, ucs_string_buffer_t *strb);
 ssize_t ucs_string_find_in_list(const char *str, const char **string_list,
                                 int case_sensitive);
 
+/**
+ * Find a string in an array of strings with NULL entries.
+ *
+ * @param str          String to search for.
+ * @param string_list  Array of strings.
+ * @param num_of_args  Number of strings in the array.
```

Comment:
I would propose to name this arg similar to array, eg `str_array` and `str_array_size`

Reply:
done